src/test/java/com/klarna/hiverunner/examples/HelloHiveRunnerParamaterizedTest.java

```diff
@@ -1,6 +1,6 @@
 /**
  * Copyright (C) 2013-2021 Klarna AB
- * Copyright (C) 2021-2024 The HiveRunner Contributors
+ * Copyright (C) 2021 The HiveRunner Contributors
```

Comment:
Once this is merged I'll take a look at the copyright plugin and regenerate all the headers.